libnavigation-ui/src/main/java/com/mapbox/navigation/ui/utils/CompareUtils.kt

```diff
@@ -0,0 +1,14 @@
+package com.mapbox.navigation.ui.utils
+
+object CompareUtils {
+    fun <T> areEqualContentsIgnoreOrder(first: Collection<T>, second: Collection<T>): Boolean {
```

Comment:
Can we make this Java user-friendly? `@JvmStatic` annotation should do the trick (👀 https://kotlinlang.org/docs/reference/java-to-kotlin-interop.html#static-methods). That way we can remove the need of having to add `INSTANCE` from Java https://github.com/mapbox/mapbox-navigation-android/blob/e6a9cf2dd6653f264026fedd551e11caa2e2162b/libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/NavigationMapRoute.java#L214

Reply:
done. this PR was already merged into master. Please see https://github.com/mapbox/mapbox-navigation-android/pull/2647